cassandra/src/main/scala/com/stratio/crossdata/connector/cassandra/CassandraQueryProcessor.scala

```diff
@@ -67,12 +90,18 @@ class CassandraQueryProcessor(cassandraRelation: CassandraXDSourceRelation, logi
 
   def execute(): Option[Array[Row]] = {
     try {
-      validatedNativePlan.map { case (columnsRequired, filters, limit) =>
-        val cqlQuery = buildNativeQuery(cassandraRelation.tableDef.name, columnsRequired, filters, limit.getOrElse(CassandraQueryProcessor.DefaultLimit))
+      validatedNativePlan.map { case (columnsRequired, filters, udfs: Map[Attribute, NativeUDF], limit) =>
+        val cqlQuery = buildNativeQuery(
+          cassandraRelation.tableDef.name,
+          columnsRequired.map(_.toString),
+          filters,
+          limit.getOrElse(CassandraQueryProcessor.DefaultLimit),
+          udfs map { case (k,v) => k.toString -> v}
+        )
         val resultSet = cassandraRelation.connector.withSessionDo { session =>
           session.execute(cqlQuery)
         }
-        sparkResultFromCassandra(columnsRequired, resultSet)
+        sparkResultFromCassandra(columnsRequired.map(_.name), resultSet)
```

Comment:
Can we retrieve multiple duplicated UDFs? I mean, execute a SELECT now(), now() FROM _;